resources/Fork.RI

```diff
@@ -13,14 +13,21 @@ UNAME=$(uname -a)
 
 if [[ $UNAME == *icrosoft* ]]; then
     # in a wsl shell
-    WSL_BUILD=$(sed -E 's/^.+-([0-9]+)-Microsoft.*/\1/' <<< $UNAME)
-    if [ -z $WSL_BUILD ]; then
+    WSL_BUILD=$(cmd.exe /c "systeminfo" | grep -E -i "build [0-9]+" | sed -E 's/^.*build ([0-9]+).*$/\1/')
```

Comment:
The sed matching needs to be case insensitive as well. The output of the command seems to depend on the locale, for me it contains `Build 19044` (with uppercase `B`).

The last part should therefore be `sed -E 's/^.*build ([0-9]+).*$/\1/I` (with `I` after the last `/`)